beluga_ros/tools/conversion_utils.py

```diff
@@ -0,0 +1,273 @@
+# Copyright 2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import h5py
+import yaml
+from typing import Dict, Optional
+from pathlib import Path
+import matplotlib.pyplot as plt
+from scipy.stats import multivariate_normal
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class DiscreteCell:
+    x: int
+    y: int
```

Comment:
@serraramiro1 meta: did you consider using a [`namedtuple`](https://docs.python.org/3/library/collections.html#collections.namedtuple)? May be simpler to use with `numpy`.

Reply:
I'd like the explicit conversion for this case